docs/siem/siem-ui.asciidoc

```diff
@@ -50,6 +50,22 @@ or the Detections API.
 [role="screenshot"]
 image::siem/images/detections-ui.png[]
 
+[float]
+[[cases-ui]]
+=== Cases (Beta)
+
+Cases are used to open and track security issues directly in the {siem-app}. 
```

Comment:
We try to avoid using the word app in the Kibana docs and use the name alone, in this case, SIEM.